android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/WalletUtils.java

```diff
@@ -40,10 +43,18 @@ private static void getPackageToBind() {
   }
 
   private static String chooseServiceToBind(List<String> packageNameServices) {
-    String[] packagesOrdered = BuildConfig.SERVICE_BIND_LIST.split(",");
-    for (String address : packagesOrdered) {
-      if (packageNameServices.contains(address)) {
-        return address;
+    if (isAppInstalled(BuildConfig.CAFE_BAZAAR_PACKAGE_NAME, context.getPackageManager())
```

Comment:
shouldn't we check if the user if from Iran **and** has cafe bazar ? because if its a a user in Europe and it has  cafe bazaar installed it will always be called over Aptoide

Reply:
That's one of the conditions